website/docs/r/virtual_machine_scale_set.html.markdown

```diff
@@ -346,6 +346,27 @@ The following arguments are supported:
 
 `storage_profile_image_reference` supports the following:
 
+* `id` - (Optional) Specifies the ID of the (custom) image to use to create the virtual 
+machine scale set, for example:
```

Comment:
I gave it a shot based on how I understood it - take a look and let me know if that's not what you had in mind.